src/style.css

```diff
@@ -1,3 +1,89 @@
 body {
+  background-color: #ddd;
+  color: black;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  /* stylelint-disable-next-line color-hex-case */
+  --blue-color-for-link: #00ACDC;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.img__link {
+  display: flex;
+  margin: 0;
+  text-decoration: none;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+:not(:first-of-type).nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 40px;
+  padding: 10px 0;
```

Comment:
better to set just `height: 60px` in such case

Reply:
Done!